acm/templates/policies/ocp-gitops-policy.yaml

```diff
@@ -195,6 +195,11 @@ spec:
                         memory: 128Mi
                     route:
                       enabled: true
+                      {{- if and (.Values.global.argocdServer) (.Values.global.argocdServer.route) (.Values.global.argocdServer.route.tls) }}
+                      tls:
+                        insecureEdgeTerminationPolicy: {{ default "Redirect" $.Values.global.argocdServer.route.tls.insecureEdgeTerminationPolicy }}
+                        termination: {{ default "passthrough" $.Values.global.argocdServer.route.tls.termination }}
```

Comment:
In the operator we went for the reencrypt default, so we should do the same here?

Reply:
@mbaldessari  Sure. Let me change that for consistency.

Reply:
@mbaldessari Pushed the changes.